src/sentry/api/bases/organization.py

```diff
@@ -238,7 +238,7 @@ def convert_args(self, request, organization_slug, *args, **kwargs):
 
         # Track the 'active' organization when the request came from
         # a cookie based agent (react app)
-        if request.auth is None and request.user:
+        if request.auth is None and request.user and not is_active_superuser(request):
```

Comment:
Does this mean that superusers will never have an `activeorg` saved even if they are a member of the org?

Reply:
Yeah, we could add a check for the org memberships here but I wasn't sure it's necessary since superuser usage should be done in rare cases and has pretty short timeout

Reply:
Lets document this assumption with a comment